week4/src/pages/mypage/mypage.tsx

```diff
@@ -0,0 +1,19 @@
+/** @jsxImportSource @emotion/react */
+import { useState } from "react";
+import Header from "../../components/mypage/header";
+import Hobby from "../../components/mypage/hobby";
+import MyProfile from "../../components/mypage/myprofile";
+
+const MyPage = () => {
+  const [currentTab, setCurrentTab] = useState("hobby");
```

Comment:
```suggestion
  const [currentTab, setCurrentTab] = useState<"hobby" | "profile">("hobby");
```

와 같은 방식으로 특정 문자열만 올 수 있는 state의 경우 타입을 조금 더 구체적으로 지정해주면 더 안전한 코드를 작성할 수 있을 것 같아요!